### PR TITLE
Add teams and users

### PR DESCRIPTION
**Pull request by sorovlad**
Add teams and subscriptions used with them.
https://github.com/moira-alert/moira/issues/230

### CODE REVIEW

**Comment by sashasushko** [comment]
@sorovlad need you my review?

**Comment by sorovlad** [comment]
@sashasushko now can you review it?

**Comment by sorovlad** [comment]
@sashasushko Sasha, can you review it now?

**Comment by sashasushko** [comment]
@sorovlad Yep. I'll do it. I need few days.

File: src/Containers/SettingsContainer.tsx [hunk 1]
```diff
@@ -252,25 +295,56 @@ class SettingsContainer extends React.Component<Props, State> {
...
+            }
         } catch (error) {
             this.setState({ error: error.message });
         } finally {
             this.setState({ loading: false });
         }
+    };
+
+    private async getTeamsAndTags() {
+        const user = await this.props.moiraApi.getUser();
```
Comment (sashasushko) [inline src/Containers/SettingsContainer.tsx:317]: `Promise.all`?

File: src/Components/Teams/Teams.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    const [addingTeam, setAddingTeam] = useState(false);
+
+    const handleAddTeam = (team: Partial<Team>) => {
+        props.addTeam(team);
+        setAddingTeam(false);
+    };
+
+    return (
+        <>
+            <div>
```
Comment (sashasushko) [inline src/Components/Teams/Teams.tsx:33]: Кажется, можно без этого `div` обойтись

**Review by sashasushko — APPROVED** [review]

**Comment by sorovlad** [comment]
Thanks, Sasha! I fixed found problems, might you approve pull request again.

**Review by sashasushko — APPROVED** [review]

**Pull request merged**